app/util/approve.py

```diff
@@ -32,95 +31,48 @@ def __init__(self):
 
     def provision_infra(member_id, user_data=None):
         # Log into OpenStack
-        conn = openstack.connect(cloud="hackucf_infra")
-
-        try:
-            os.remove("terraform.tfstate")
-        except Exception:
-            pass
-
-        try:
-            os.remove("terraform.tfstate.backup")
-        except Exception:
-            pass
-
         try:
             with Session(engine) as session:
                 if not user_data:
                     user_data = UserModel(
```

Comment:
I feel like this should be a function call

Reply:
Probably all the user database calls should be abstracted away to allow caching, for now I just required user_data to be passed to this function